tests/Oryx.Integration.Tests/Oryx.Integration.Tests.csproj

```diff
@@ -20,6 +20,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\src\BuildScriptGeneratorCli\BuildScriptGeneratorCli.csproj" />
```

Comment:
To access constants from product code...otherwise the magic strings are getting out of hand and it is easy to make error